docs/pages/versions/unversioned/workflow/development-mode.md

```diff
@@ -1,26 +1,54 @@
 ---
-title: Development Mode
+title: Development and Production Mode
 old_permalink: /versions/v12.0.0/guides/development-mode.html
 previous___FILE: ./up-and-running.md
 next___FILE: ./configuration.md
 ---
 
-React Native includes some very useful tools for development: remote JavaScript debugging in Chrome, live reload, hot reloading, and an element inspector similar to the beloved inspector that you use in Chrome. It also performs bunch of validations while your app is running to give you warnings if you're using a deprecated property or if you forgot to pass a required property into a component, for example.
+import Video from '../../../../components/plugins/Video'
 
-![Screenshots of development mode in action](/static/images/development-mode.png)
+Your project will always run in one of two modes: development or production. By default, running your project locally with `expo start` runs it in development mode, whereas a published project (via `expo publish`), or any standalone apps, will run in production mode.
+
+The difference between these is pretty simple- production mode minifies your code and better represents the performance your app will have on end users' devices, whereas development mode gives you have access to tools that make development and **especially** debugging a lot easier. We go through those tools and how to use them in the [debugging documention](../debugging/), so here we'll just describe some of the use cases and differences, as well as how to switch between the two modes.
```

Comment:
we should avoid saying things are simple/easy - https://css-tricks.com/words-avoid-educational-writing/

instead, we can just get to the point. 

```suggestion
Production mode [minifies your code](link-to-something-about-minification-here) and better represents the performance your app will have on end users' devices. Development mode includes useful warnings and gives you access to tools that make development and debugging easier. Let's look at each of these modes more in detail and learn how you can toggle between them.
```